packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -463,12 +500,22 @@ class SliverGeometry extends Diagnosticable {
   /// position by default.
   final double paintOrigin;
 
-  /// The amount of visual space that was taken by the sliver to render the
-  /// subset of the sliver that covers all or part of the
-  /// [SliverConstraints.remainingPaintExtent].
+  /// The amount of currently visible visual space that was taken by the sliver
+  /// to render the subset of the sliver that covers all or part of the
+  /// [SliverConstraints.remainingPaintExtent] in the current viewport.
+  ///
+  /// This value does not affect how the next sliver is positioned. In other
+  /// words, if this value was 100 and [layoutExtent] was 0, a typical sliver
+  /// placed after it would overlap while painting by 100 pixel directly on
+  /// top of this sliver.
```

Comment:
actually iirc the paint order would put subsequent slivers underneath, not over, this one

Reply:
Ah ok. Just removed the ordering. It wasn't necessary.